test/wpt/status/streams.json

```diff
@@ -54,10 +54,44 @@
       ]
     }
   },
+  "transform-streams/cancel.any.js": {
+    "fail": {
+      "expected": [
+        "cancelling the readable side should call transformer.cancel()",
+        "cancelling the readable side should reject if transformer.cancel() throws",
+        "aborting the writable side should call transformer.abort()",
+        "aborting the writable side should reject if transformer.cancel() throws",
+        "closing the writable side should reject if a parallel transformer.cancel() throws",
+        "readable.cancel() and a parallel writable.close() should reject if a transformer.cancel() calls controller.error()",
+        "writable.abort() and readable.cancel() should reject if a transformer.cancel() calls controller.error()"
+      ]
+    }
+  },
+  "transform-streams/errors.any.js": {
+    "fail": {
+      "expected": [
+        "controller.error() should close writable immediately after readable.cancel()"
+      ]
+    }
+  },
+  "transform-streams/general.any.js": {
+    "fail": {
+      "expected": [
+        "terminate() should abort writable immediately after readable.cancel()"
+      ]
+    }
+  },
   "transform-streams/invalid-realm.tentative.window.js": {
     "skip": "Browser-specific test"
   },
   "readable-streams/read-task-handling.window.js": {
     "skip": "Browser-specific test"
+  },
+  "transform-streams/reentrant-strategies.any.js": {
+    "fail": {
+      "expected": [
+        "writer.abort() inside size() should work"
+      ]
+    }
```

Comment:
Hmm, not sure about this one, will investigate further. 🕵️

Reply:
I think this one is also related to https://github.com/nodejs/node/pull/50126 somehow I believe this was changed for the .cancel impl saw this one failing too on my PR